.github/workflows/schema-validation.yml

```
@@ -0,0 +1,76 @@
+          POSTGRES_DB: test_db
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          - 5432:5432
+
+    steps:
+      - uses: actions/checkout@v4
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)